test/e2e/aws_test.go

```diff
@@ -164,29 +161,30 @@ func makeCluster() *capi.Cluster {
 }
 
 func makeMachine() *capi.Machine {
-	yaml, err := ioutil.ReadFile(*machineYAML)
-	Expect(err).To(BeNil())
+	// yaml, err := ioutil.ReadFile(*machineYAML)
```

Comment:
maybe delete instead of comment?

Reply:
I commented it out because it should be fixed in another iteration, it's helpful to keep this around for reference, same happened in CAPI before the logic went in